cmd/traffic/cmd/manager/managerutil/envconfig.go

```diff
@@ -52,6 +52,7 @@ type Env struct {
 
 	AgentRegistry             string                      `env:"AGENT_REGISTRY,           parser=nonempty-string"`
 	AgentImage                string                      `env:"AGENT_IMAGE,              parser=string,         default="`
+	AgentImagePullPolicy      string                      `env:"AGENT_IMAGE_PULL_POLICY,  parser=string,		 default="`
```

Comment:
```suggestion
	AgentImagePullPolicy      string                      `env:"AGENT_IMAGE_PULL_POLICY,  parser=string,         default="`
```
because using tabs inside a string makes the string misaligned unless you have a specific tab-with setting.